nbodykit/measurestats.py

```diff
@@ -481,7 +481,7 @@ def project_to_basis(comm, x3d, y3d, edges, los='z', poles=[], symmetric=True):
                     weighted_y3d.real[:, nonsingular] *= 2.
                     weighted_y3d.imag[:, nonsingular] = 0.
                     
-            weighted_y3d *= 0.5*(2*ell + 1.)
+            weighted_y3d *= (2.*ell + 1.)
```

Comment:
If you mean we are double counting the number of modes at line 460, then yes.

But the number of modes also included non-independent modes. so sqrt(Nmodes) is no longer the error. Right?